modules/creating-an-accelerator-profile.adoc

```diff
@@ -7,45 +7,47 @@
 +
-. Click the `AcceleratorProfile` custom resource definition (CRD).
+The *Create accelerator profile* dialog appears.
+. In the *Name* field, enter an appropriate name for the accelerator profile.
+. In the *Identifier* field, enter a unique string that identifies the hardware accelerator associated with the accelerator profile.
+. Optional: In the *Description* field, enter an appropriate description for the accelerator profile.
+. To enable or disable the accelerator profile immediately after creation, click the toggle in the *Enable* column.
+. Optional: Add a toleration to schedule pods with matching taints.
+.. Click *Add toleration*. 
 +
-A details page for the custom resource definition (CRD) opens.
-. Click the *Instances* tab.
-. Click *Create AcceleratorProfile*. 
-+ 
-The *Create AcceleratorProfile* page opens with an embedded YAML editor. 
-. Enter or paste your accelerator profile YAML code directly in the embedded editor. This example shows the code for a Habana Gaudi 1 accelerator profile:
-+
-[source,yaml]
----
-apiVersion: dashboard.opendatahub.io/v1alpha
-kind: AcceleratorProfile
-metadata:
-  name: hpu-profile-first-gen-gaudi
-spec:
-  displayName: Habana HPU - 1st Gen Gaudi
-  description: First Generation Habana Gaudi device
-  enabled: true
-  identifier: habana.ai/gaudi
-  tolerations:
-    - effect: NoSchedule
-      key: habana.ai/gaudi
-      operator: Exists
----
-. Click *Create*.
+The *Add toleration* dialog appears.
+.. From the *Operator* list, select one of the following options:
+* *Equal* - The *key/value/effect* parameters must match. This is the default.
+* *Exists* - The *key/effect* parameters must match. You must leave a blank value parameter, which matches any.
+.. From the *Effect* list, select one of the following options:
+* *None* 
+* *NoSchedule* - New pods that do not match the taint are not scheduled onto that node. Existing pods on the node remain.
+* *PreferNoSchedule* - New pods that do not match the taint might be scheduled onto that node, but the scheduler tries not to. Existing pods on the node remain.
+* *NoExecute* - New pods that do not match the taint cannot be scheduled onto that node. Existing pods on the node that do not have a matching toleration are removed.
+.. In the *Key* field, enter a toleration key. The key is any string, up to 253 characters. The key must begin with a letter or number, and may contain letters, numbers, hyphens, dots, and underscores.
+.. In the *Value* field, enter a toleration value. The value is any string, up to 63 characters. The value must begin with a letter or number, and may contain letters, numbers, hyphens, dots, and underscores.
+.. In the *Toleration Seconds* section, select one of the following options to specify how long a pod stays bound to a node that has a node condition. 
+** *Forever* - Pods stays permanently bound to nodes. 
+** *Custom value* - Enter a value (in seconds) before pods can remain bound to a nodes.
```

Comment:
- Looks like 'nodes' should be singular here.
- We can remove the parentheses.

- Unsure on phrasing of 'before pods can remain bound to a nodes'. Does the custom value define how long pods will stay bound to a node with a node condition before being released?

Reply:
Per chat, suggesting a reword for everywhere we have this line (45):

_Enter a value in seconds to define how long pods stay bound to a node that has a node condition._

Reply:
Changed to your suggestion.